backend/app/api/endpoints/internal/conversion_callback.py

```
@@ -0,0 +1,138 @@
+
+router = APIRouter(
+    prefix="/conversion/callback",
+    tags=["conversion-callback"],
+    dependencies=[Depends(verify_internal_service_token)],
+)
+
+
+@router.post("/status", response_model=ConversionStatusResponse)
+def conversion_status_callback(
+    request: ConversionStatusRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)